tests/examples/gettingStarted.t

```diff
@@ -73,6 +73,7 @@ my $mobileUserAgent = 'Mozilla/5.0 (iPhone; CPU iPhone OS 7_1 like Mac OS X) App
 my $r = get_with_ua('/hash', $mobileUserAgent);
 like($r, qr/x-mobile: True/, 'Mobile match (single User-Agent)');
 like($r, qr/x-device: [a-zA-Z ]+,[a-zA-Z ]+/, 'Mobile match (single User-Agent)');
+like($r, qr/x-device-hinted: [a-zA-Z ]+,[a-zA-Z ]+/, 'Mobile match (single User-Agent)');
```

Comment:
can we provide evidence consisting not of a single user agent, but of client hints for the example that relies on client hints?

Reply:
it would be good to add an example that without client hints would otherwise match something wrong - f.e. client hints should contain platform version and model and we should check that the result contains those

Reply:
* https://github.com/51Degrees/device-detection-nginx/pull/171/files#diff-622d9c2bd70ea4b2d3053757e02f06c4b273cbf36afda6695c41e0201e99d4b8R378-R382
* https://github.com/51Degrees/device-detection-nginx/pull/171/files#diff-622d9c2bd70ea4b2d3053757e02f06c4b273cbf36afda6695c41e0201e99d4b8R372-R376
* https://github.com/51Degrees/device-detection-nginx/pull/171/files#diff-622d9c2bd70ea4b2d3053757e02f06c4b273cbf36afda6695c41e0201e99d4b8R366-R370
* https://github.com/51Degrees/device-detection-nginx/pull/171/files#diff-622d9c2bd70ea4b2d3053757e02f06c4b273cbf36afda6695c41e0201e99d4b8R360-R364
* https://github.com/51Degrees/device-detection-nginx/pull/171/files#diff-622d9c2bd70ea4b2d3053757e02f06c4b273cbf36afda6695c41e0201e99d4b8R295-R310